bindings/wasm/examples/src/explorer-did.js

```diff
@@ -0,0 +1,243 @@
+        type: "LinkedDomains",
+        serviceEndpoint: "https://iota.org",
+    };
+    diffDoc1.insertService(Service.fromJSON(serviceJSON1));
+    diffDoc1.metadataUpdated = Timestamp.nowUTC();
+
+    // Create a signed diff update.
+    //
+    // This is the first diff so the `previousMessageId` property is
+    // set to the last DID document published on the integration chain.
+    const diff1 = intDoc2.diff(diffDoc1, intReceipt2.messageId, key, intDoc2.defaultSigningMethod().id.toString());
+
+    // Publish the diff to the Tangle, starting a diff chain.
+    const diffReceipt1 = await client.publishDiff(intReceipt2.messageId, diff1);
+
+
+    // ===========================================================================
+    // Spam on first diff update.
+    // ===========================================================================
+
+    let diffIndex = Document.diffIndex(intReceipt2.messageId);
+    await client.publishJSON(diffIndex, { "diffSpam:1": true });
+    await client.publishJSON(diffIndex, { "diffSpam:2": true });
+
+    // ===========================================================================
+    // Diff Chain Update 2
+    // ===========================================================================
+
+    console.log("start diff update 2");
+
+    // Prepare another diff chain update.
+    const diffDoc2 = Document.fromJSON(diffDoc1.toJSON());
+
+    // Add a second Service with the tag "linked-domain-2"
+    let serviceJSON2 = {
+        id: diffDoc2.id + "#linked-domain-2",
+        type: "LinkedDomains",
+        serviceEndpoint: {
+            "origins": ["https://iota.org/", "https://example.com/"]
+        },
+    };
+    diffDoc2.insertService(Service.fromJSON(serviceJSON2));
+    diffDoc2.metadataUpdated = Timestamp.nowUTC();
+
+    // This is the second diff therefore its `previousMessageId` property is
+    // set to the first published diff to extend the diff chain.
+    const diff2 = diffDoc1.diff(diffDoc2, diffReceipt1.messageId, key, diffDoc1.defaultSigningMethod().id.toString());
+
+    // Publish the diff to the Tangle.
+    // Note that we still use the `messageId` from the last integration chain message here to link
+    // the current diff chain to that point on the integration chain.
+    const diffReceipt2 = await client.publishDiff(intReceipt2.messageId, diff2);
+
+
+
+    // ===========================================================================
+    // Diff Chain Update 3
+    // ===========================================================================
+
+    console.log("start diff update 3");
+
+    // Prepare another diff chain update.
+    const diffDoc3 = Document.fromJSON(diffDoc2.toJSON());
+
+    // Add a second Service with the tag "linked-domain-2"
+    let serviceJSON3 = {
+        id: diffDoc3.id + "#linked-domain-3",
+        type: "LinkedDomains",
+        serviceEndpoint: {
+            "origins": ["https://iota.org/", "https://example.com/"]
+        },
+    };
```

Comment:
Opinion: maybe this should use the set syntax for the `serviceEndpoint` to differentiate it from the second service example?

```suggestion
    // Add a second Service with the tag "linked-domain-2"
    let serviceJSON3 = {
        id: diffDoc3.id + "#linked-domain-3",
        type: "LinkedDomains",
        serviceEndpoint: ["https://iota.org/", "https://example.com/"],
    };
```